### DISCUSSION TITLE
Currency Converter using Google :)

### DISCUSSION THREAD

**Discussion by hakubo** [johnlindquist/kit#494] [Share]
Since most of currency converters are paid APIs - why not just use google for this task :)

https://user-images.githubusercontent.com/1018759/138569135-4b682b44-8b8b-4e9b-bd30-64c180bd1d5f.mp4

[Install currency-converter](https://scriptkit.com/api/new?name=currency-converter&url=https://gist.githubusercontent.com/hakubo/d4822cfc6e42d7dead70d4da73bfded9/raw/97afb98be93518b2550448e5b81d425b79642ed2/currency-converter.js")

```js
// Menu: Currency Converter using google
// Author: Jakub Olek
// Twitter: @JakubOlek

/** @type {import("@johnlindquist/kit")} */

const from = await arg("from", ["pln", "usd", "eur"]);
const to = await arg("to", ["usd", "pln", "eur"]);
const query = await arg("amount");
const value = await scrapeSelector(
  `https://google.com/search?q=${encodeURIComponent(
    query + " " + from + " " + to
  )}`,
  "span[data-value]"
);

div(query + " " + from + " = " + value + " " + to, "p-4");

```

**Comment by johnlindquist** [comment]
Lol, awesome. scrapeSelector is way too much fun 😊
